### PR TITLE
closes issue #74

### PR DESCRIPTION
**Pull request by czucca**
Added condition "bold != .05 &&" to` internal.R` lines 676 and 757 in order to mute the star, if bold = .05 in `texreg` and `htmlreg` when CI are displayed.

### CODE REVIEW

File: R/internal.R [hunk 1]
```diff
@@ -689,7 +689,7 @@ outputmatrix <- function(m, single.row, neginfstring, posinfstring,
                                        ci = ci
                         )$coefficients
                     } else { # significance from confidence interval
-                        if (is.numeric(ci.test) && !is.na(ci.test) && 
+                        if (is.numeric(ci.test) && !is.na(ci.test) && bold != .05 &&
```
Comment (leifeld) [inline R/internal.R]: The `bold` argument can take other values than 0.05. If the user sets `bold = 0.01`, you would probably not want stars to be printed either, right? So I would do something like `bold == 0`.

File: R/internal.R [hunk 2]
```diff
@@ -782,7 +782,7 @@ outputmatrix <- function(m, single.row, neginfstring, posinfstring,
                                        star.suffix = star.suffix
                         )$coefficients
                     } else { # significance from confidence interval
-                        if (is.numeric(ci.test) && !is.na(ci.test) && 
+                        if (is.numeric(ci.test) && !is.na(ci.test) && bold != .05 &&
```
Comment (leifeld) [inline R/internal.R]: Same as above.

**Comment by czucca** [comment]
yes! tested it and good to go! Have you unblocked me? I see that I can merge it.

**Comment by leifeld** [comment]
No, that's because I have approved the changes already... you can squash and merge it if you want.

**Comment by czucca** [comment]
I'm happy like a kid pushing the elevator button! :)

**Pull request merged**